server/Routes/authRoutes.js

```diff
@@ -1,5 +1,50 @@
-// const express = require('express');
+const express = require('express');
+const passport = require('passport');
 
-// const router = express.Router();
+const router = express.Router();
 
-// module.exports = router;
+router.get('/google', passport.authenticate('google', {
+  // as a part of the passport middleware process, we are expecting the profile info back from Google
+  scope: ['profile'],
+}));
+
+router.get('/google/redirect', (req, res, next) => {
+  passport.authenticate('google', (err, user) => {
+    // if passport authenticate errors out
```

Comment:
Haven't seen routes as good as these in some time. Outstanding!

Reply:
@mitchelsevere- router extraordinaire 🚧